frame/aura/src/lib.rs

```diff
@@ -165,7 +187,7 @@ impl<T: Config> sp_runtime::BoundToRuntimeAppPublic for Pallet<T> {
 	type Public = T::AuthorityId;
 }
 
-impl<T: Config> OneSessionHandler<T::AccountId> for Pallet<T> {
+impl<T: Config> OnSessionHandler<T::AccountId, T::MaxAuthorities> for Pallet<T> {
```

Comment:
I think this was supposed to be called like this, `One-Session-Handler`, a single session handler, and `pallet-session` is given a tuple of `OneSessionHandler`s.

Reply:
Really though? I think OnSessionHandler makes sense cause this is all about what happens on a session?

haha i really dunno